docs/jdbc.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
💯

Reply:
Haha can't get me this time

Reply:
Oh shoot I need to update the file name too